app/build.gradle

```diff
@@ -117,4 +124,5 @@ dependencies {
   androidTestImplementation rootProject.ext.facebookSnapshotTestingCore
   implementation project(':Backpack')
   implementation project(':backpack-compose')
+  ksp project(":ksp")
```

Comment:
can we give the module a more descriptive name? `ksp` is more the implementation detail, rather than what it actually does and if we ever need another `ksp` implementation we'll get a clash